src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -40,10 +34,66 @@ public static void Run(ConfigPathsArgs args, Func<ILogger> getLogger)
         }
     }
 
+    internal static class ConfigGetRunner
+    {
+        public static void Run(ConfigGetArgs args, Func<ILogger> getLogger)
+        {
+            RunnerHelper.ValidateArguments(args, getLogger);
+
+            if (args.AllOrConfigKey == null)
+            {
+                throw new CommandException(string.Format(CultureInfo.CurrentCulture, Strings.ConfigCommandKeyNotFound, args.AllOrConfigKey));
+            }
+
+            var settings = RunnerHelper.GetSettingsFromDirectory(args.WorkingDirectory) as Settings;
+            if (settings == null)
+            {
+                return;
+            }
+            ILogger logger = getLogger();
```

Comment:
Just FYI something I checked: The logger returned by `getLogger` in theory could still be null. But from a quick check, I don't think it's possible since our app initialization always logic creates one.